src/library/more-instances.lisp

```diff
@@ -0,0 +1,13 @@
+(in-package #:coalton-library)
+
+(coalton-toplevel
```

Comment:
Is there a reason this can't be in classes.lisp?

Reply:
`not` wasn't defined earlier enough. Now it's a non-problem.